modules/qtwidgets/properties/transferfunctionpropertydialog.cpp

```diff
@@ -200,6 +254,36 @@ TransferFunctionPropertyDialog::TransferFunctionPropertyDialog(TransferFunctionP
 
     setWidget(mainPanel);
 
+    // set up color dialog
+    {
+        colorDialog_ = util::make_unique<QColorDialog>(this);
+        colorDialog_->hide();
+        // we don't want to see alpha in the color dialog
+        colorDialog_->setOption(QColorDialog::ShowAlphaChannel, false);
+        colorDialog_->setOption(QColorDialog::NoButtons, true);
+        colorDialog_->setWindowModality(Qt::NonModal);
+        colorDialog_->setWindowTitle(QString("TF Primitive Color - %1")
+                                         .arg(utilqt::toQString(tfProperty_->getDisplayName())));
+
+        connect(tfEditor_.get(), &TransferFunctionEditor::showColorDialog,
```

Comment:
There used to be some mac bug workaroud here right?

Reply:
added MacOS-specific fix